sign/eddsa/eddsa_test.go

```diff
@@ -101,6 +101,35 @@ func TestEdDSASigning(t *testing.T) {
 	}
 }
 
+// Test signature malleability
+func TestEdDSASigningMalleability(t *testing.T) {
+	/* 2^252+27742317777372353535851937790883648493 */
```

Comment:
is this the prime order of the base point? That would be more interesting to know here - I have to admit I only recognize the 2**252, which I know is part of the prime order. But I don't know if it's +1, -1, +1234567...

Reply:
Yes, that is l, the prime order of the base point: I'll document it on the source code.